src/components/misc/CorruptedStateDialog.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+    <b-modal
+        id="corrupted-state-dialog"
+         ref="corruptedModalRef"
+        header-class="title-header"
+        footer-class="title-footer"
+    >
+        <div>
+            <label> We are facing some issues, please reload this page</label>
+        </div>
+        <template v-slot:modal-footer>
+            <div class="w-100">
+                <b-button variant="outline-primary" @click="reloadPage()"
```

Comment:
Very nice!

Reply:
Yes, we should start emitting this event soon.